style.css

```diff
@@ -240,3 +249,233 @@ ul .languages {
 }
 
 .card2_content li:hover { background-color: #344563; }
+
+/* Contact Form */
+
+.contact_form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url("./images/illustration-contact-form.svg") no-repeat;
+  background-size: 10%;
+  background-position: bottom 275px right 0;
+  width: 100%;
+}
+
+.send_me_msg {
+  text-align: center;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 40px;
+  margin: 60px 24px 58px 24px;
+}
+
+.form_items {
```

Comment:
- [ ] ***Make sure that the students Desktop version looks like the one in the design they chose (*TemplateX-main* file). Check if the website changes while switching from mobile to desktop*** is one of the requirements but I can see your mobile version is broken, KIndly fix your mobile version as the previous reviewer suggested,

Reply:
I don't get what's the issue you are trying to say here. The previous reviewer said that the cards of the projects and the about section are aligned to the left so I fixed it and it looks good, all sections are centered like the design.

![Screenshot 2021-12-03 135012](https://user-images.githubusercontent.com/92228303/144605550-9d71cd86-d9b7-4428-85cb-a29b4a2d7a26.png)

![Screenshot 2021-12-03 134946](https://user-images.githubusercontent.com/92228303/144605588-30393ee6-a648-4a9e-bbf2-370a70cf2ecb.png)

![Screenshot 2021-12-03 135027](https://user-images.githubusercontent.com/92228303/144605599-cd5725d1-57c2-49d0-be52-eb7bc72b247f.png)